src/pages/Calendar.css

```
@@ -0,0 +1,334 @@
+  font-size: 20px;
+  font-weight: 800;
+  margin: 0;
+  min-width: 130px;
+  text-align: center;
+}
+
+.calendar-toolbar ion-button.calendar-nav-btn {
+  --color: var(--text-secondary);
+  --padding-start: 8px;
+  --padding-end: 8px;
+  margin: 0;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)